components/mpas-seaice/src/shared/mpas_seaice_initialize.F

```diff
@@ -558,6 +564,124 @@ subroutine init_ice_state_uniform_ice(&
 
   end subroutine init_ice_state_uniform_ice!}}}
 
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+!
+!  init_ice_state_uniform_1D
```

Comment:
Do thermodynamic tracer values not need to be set by this testcase with colpkg_init_trcr (e.g. https://github.com/E3SM-Project/E3SM/blob/master/components/mpas-seaice/src/shared/mpas_seaice_initialize.F#L705?) It seems from the IPSOL namelist file that vertical thermodynamics is active so would need the tracers initialized?

Reply:
@akturner :  Yes, thanks.  I added the active tracer initializations and re-tested.